client/src/components/ShoeTile.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import { Link } from "react-router-dom"
+
+const ShoeTile = (props) => {
+    let showId = `/${props.id}`
+    return (
+        <div className="callout">
+            <img src={props.url} width="200" height="200"/>
```

Comment:
styling and formatting should be done with CSS so that it is all managed in one place, rather than some styling done inline and some in CSS files